app/views/admin/dashboard/show_users.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>All Users</h1>
+<ul>
+  <% @users.each do |user| %>
```

Comment:
Do we still want our @users being referenced here or do we want to reference our `current_user` helper method?

Reply:
This show page is specific to making sure that the admin can easily see who has access to what permissions within the app.  My apologies for missing that detail, disregard my comment about current_user.